docs/src/components/Footer/AvatarList.astro

```diff
@@ -1,6 +1,12 @@
 ---
 // fetch all commits for just this page's path
-const path = "docs/" + Astro.props.path;
+
+export interface Props {
+  path: string;
+}
+
+const props = Astro.props.path as Props;
```

Comment:
for here and any other similar spots: can you convert this to the format `const {path} = Astro.props as Props;`? This is the pattern that we promote and I think people look to these examples in our repo for guidance.